source/basic.tex

```diff
@@ -3221,7 +3221,7 @@
 
 struct A { unsigned char a[32]; };
 struct B { unsigned char b[16]; };
-A a;
+alignas(int) A a;
 B *b = new (a.a + 8) B;                 // \tcode{a.a} provides storage for \tcode{*b}
```

Comment:
This used to assume that `a.a + 12` is `int`-aligned. Now it assumes that `a.a + 12` is `int`-aligned if `a.a` is `int`-aligned, i.e., that `4 % alignof(int) == 0`, this is still not a portable assumption. Should we `static_assert` that fact?

Reply:
Again, I'd prefer a different representation of such an assumption.

For example, a comment that says "assumes that a.a + 12 is suitably aligned for an `int`"

Reply:
It seems sufficient to me to say "assumes that `sizeof(int)` is equal to `4`", given that `sizeof(T)` must be a multiple of `alignof(T)`.